beluga/test/beluga/sensor/test_lfm_with_unknown_space.cpp

```diff
@@ -276,7 +276,7 @@ TEST(LikelihoodFieldModelUnknownSpace, GridWithRotationAndOffset) {
   }
 }
 
-TEST(LikelihoodFieldModelUnknownSpace, GridUpdates) {
+TEST(LikelihoodFieldModelUnknownSpace, DISABLED_GridUpdates) {
```

Comment:
I still need to figure why this test fails now.

Reply:
Fixed. The reason the test began to fail was because the behavior tested was actaully wrong. See #472 .